projects/BFDR/CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Changelog
+
+All notable changes to this project will be documented in this file. See [commit-and-tag-version](https://github.com/absolute-version/commit-and-tag-version) for commit guidelines.
+
+## [1.0.1-preview](https://github.com/nightingaleproject/vital-records-dotnet/commit/725088c0632eff716e0e865b07014af595b99ca3) (2024-04-24)
+
+
+### Features
+
+
+
+### Bug Fixes
+
+* add versin file ([58b8c1a](https://github.com/nightingaleproject/vital-record-dotnet-demo/commit/58b8c1ab7ea4fd1260ccf1d608a336d7c43a1ee3))
```

Comment:
typo in version

Reply:
Will fix. Because BFDR does not actually get released/tagged independently I am just linking to the last commit on this PR. In [VRDR](https://github.com/nightingaleproject/vrdr-dotnet/blob/master/CHANGELOG.md?plain=1) it links to an actual tagged release.